### PR TITLE
fix: tolerate partial folder reads during upload

### PR DESCRIPTION
**Pull request by rezk2ll**
## Problem

On Windows, dropping a folder where a deep path exceeds the OS length limit failed mid-walk with a `NotFoundError`. The pipeline displayed the right alert and marked the top-level placeholder as errored, but every ancestor folder up to the failure had already been created server-side. The user was left with a partial folder structure and no way to know which subtree was missing, so retrying meant guessing what was incomplete.

## Solution

- Walk the local FileSystem API tree first with no server calls; capture read failures per node instead of throwing.
- Then create the full server-side structure, including empty folders and ancestors of unreadable subtrees, so the resulting tree matches the shape that was dropped.
- Surface each failed read as its own queue row, so it's clear exactly which path was skipped.
- Tag folder failures with `isDirectory: true` so the queue renders a folder glyph for them, distinct from file failures.
- Route errors through the shared classifier: `NotFoundError` lands on `unreadable`, permission and I/O failures land on `failed`.
- Reveal the full relative path on hover via a tooltip, since long paths get ellipsised in the row.

## Changes

- Replace `flattenDirectoryEntry` with `walkDirectoryEntry` (local-only walk) and `materializeNode` (server folder creation + queue item emission); folders are always created, including empty ones.
- Add `makeFailedItem` to build queue rows for entries that couldn't be read locally, with status set via `classifyUploadError` and `isDirectory` propagated to the renderer.
- Wrap the queue row's `displayName` in a `cozy-ui` `Tooltip` (gated on `relativePath`) so deep paths are readable on hover.
- Add tests covering ancestor creation when the deepest folder is unreadable, empty-folder creation, mixed readable and unreadable siblings, the non-`NotFoundError` classification path, and the placeholder replacement flow.

## Summary by CodeRabbit

* **Bug Fixes**
  * More robust upload traversal: unreadable files/directories no longer throw during scanning, ancestor folders are still created, and unreadable items appear as distinct "unreadable" rows in the upload queue.
* **New Features**
  * Tooltips show full filenames when names are truncated in the upload queue.
* **Tests**
  * Expanded test coverage for unreadable paths, mixed readable/unreadable trees, empty directories, and placeholder resolution.

### CODE REVIEW

**Review by zatteo — APPROVED** [review]

File: src/modules/upload/index.spec.js [hunk 1]
```diff
@@ -795,6 +832,110 @@ describe('flattenEntries', () => {
...
+    const empty = createMockDirEntry('empty', [])
+    const top = createMockDirEntry('top', [empty])
+    const entries = [{ file: null, isDirectory: true, entry: top }]
+
+    const result = await flattenEntries(entries, 'root', fakeClient, null)
+
+    const createdNames = createDirectorySpy.mock.calls.map(c => c[0].name)
+    expect(createdNames).toEqual(['top', 'empty'])
+    expect(result).toEqual([])
+  })
```
[the remark this answers is not in the document]
Comment (rezk2ll) [reply src/modules/upload/index.spec.js:874]: i think you are drunk

**Review by Crash-- — APPROVED** [review]
let's see
